virtualmachine/vsphere/vm.go

```diff
@@ -785,31 +798,33 @@ func getIpFromVmMo(vmMo *mo.VirtualMachine) []net.IP {
 	return ips
 }
 
-// GetIPsAndId returns the IPs and reference Id of this VM. Returns all the IPs known to the API for
-// the different network cards for this VM. Includes IPV4 and IPV6 addresses.
-func (vm *VM) GetIPsAndId() ([]net.IP, string, error) {
+// GetIPsAndIds returns the IPs, reference Id and instance uuid of this VM.
+// Returns all the IPs known to the API for the different network cards
+// for this VM. Includes IPV4 and IPV6 addresses.
+func (vm *VM) GetIPsAndIds() ([]net.IP, string, string, error) {
```

Comment:
We are returning too many values here, should we use existing structures here. 
For eg: VMInfo